cli/templateversions.go

```diff
@@ -169,3 +170,66 @@ func templateVersionsToRows(activeVersionID uuid.UUID, templateVersions ...coder
 
 	return rows
 }
+
+func (r *RootCmd) templateVersionsPromote() *serpent.Command {
+	var (
+		templateName        string
+		templateVersionName string
+		orgContext          = NewOrganizationContext()
+	)
+	client := new(codersdk.Client)
+	cmd := &serpent.Command{
+		Use:   "promote",
```

Comment:
I don't think there is precedence for required flags, can we add to the `Use` to include the required flags?

```suggestion
		Use:   "promote --template=<template_name> --template-version=<template_version_name>",
```

We include required params in other `Use` fields: https://github.com/coder/coder/blob/61c37d039bd5f9a3fac005502d36f436a7ce2e24/cli/autoupdate.go#L18-L18